docs/_includes/js/store.js

```diff
@@ -0,0 +1,102 @@
+(function() {
+  var searchBar      = document.querySelector('.SearchBar'),
+      searchBarInput = document.querySelector('.SearchBar input'),
+      searchResults  = document.querySelector('.SearchBar-resultsData'),
+
+      dropdown        = document.querySelector('.Dropdown'),
+      dropdownLabel   = document.querySelector('.Dropdown-selection')
+      dropdownOptions = document.querySelectorAll('.Dropdown .Dropdown-options a'),
+
+      removeIcon = document.querySelector('.remove-icon');
+
+  document.addEventListener('click', function(e) {
+    if (e.target.id != "dropdown-selection" && dropdown.classList.contains('is-shown')) {
+      dropdown.classList.remove('is-shown');
+    }
+  });
+
+  dropdown.addEventListener('click', function() {
+
+    if (this.classList.contains('is-shown')) {
+      this.classList.remove('is-shown');
+    } else {
+      this.classList.add('is-shown');
+    }
+
+    return false;
+  });
+
+  [].forEach.call( dropdownOptions, function(el) {
+    el.addEventListener('click', function() {
+      dropdownLabel.innerHTML = this.innerHTML;
+
+      if (this.classList.contains('stacks')) {
+        dropdownLabel.classList.remove('stencils');
+        dropdownLabel.classList.add('stacks');
+      } else {
+        dropdownLabel.classList.remove('stacks');
+        dropdownLabel.classList.add('stencils');
+      }
+
+      searchBarInput.focus();
+    }, false)
+  });
+
+  searchBarInput.addEventListener('keyup', function(e) {
+    submitInput(this);
+  });
+
+  removeIcon.addEventListener('click', function(e) {
+    searchBarInput.value = "";
+    searchBarInput.focus();
+    searchBar.classList.remove('is-shown');
+    searchResults.innerHTML = "";
+
+    return false;
+  });
+
+  var searchResultTemplate = function(obj) {
+    return '<a class="SearchBar-resultsData-row">' +
+      '<p class="title">' + obj.formattedTitle + (obj.verified ? '<span class="u-icon u-verified"></span>' : '') + '</p>' +
+      '<p class="description">by '+ obj.author +'</p>' +
+      '</a>';
+  }
+
+  var wrapMatchedText = function(title, matchedText, openTag, closeTag) {
+    var indexOf = title.toLowerCase().indexOf(matchedText),
+        len     = matchedText.length;
+
+    var start  = title.substr(0, indexOf),
+        middle = openTag + title.substr(indexOf, len) + closeTag,
+        end    = title.substr(indexOf + len);
+
+    return start + middle + end;
+  }
+
+  var submitInput = function(e) {
```

Comment:
if you apply the comment above, `e` becomes the `event` which is what you'd expect while reading this code (and also what you'd expect in js code) and `this` becomes the `searchInput` so that you can use both if needed.
```js
var submitInput = function(e) {
  //...
  if (this.value != "") {
  //...
```